src/jobflow_remote/utils/db.py

```diff
@@ -149,25 +150,19 @@ def __init__(
 
     @classmethod
     def get_lock_time(cls, d: dict):
-        """
-        Get the time the document was locked on a dictionary.
-        """
+        """Get the time the document was locked on a dictionary."""
         return d.get(cls.LOCK_TIME_KEY)
 
     @classmethod
     def get_lock_id(cls, d: dict):
-        """
-        Get the lock id on a dictionary.
-        """
+        """Get the lock id on a dictionary."""
         return d.get(cls.LOCK_KEY)
 
-    def acquire(self):
-        """
-        Acquire the lock
-        """
+    def acquire(self) -> None:
+        """Acquire the lock."""
         # Set the lock expiration time
         now = datetime.utcnow()
-        db_filter = copy.deepcopy(self.filter)
+        db_filter = copy.deepcopy(dict(self.filter))
```

Comment:
this fixes a `mypy` issue:
```
src/jobflow_remote/utils/db.py:176: error: "Mapping[str, Any]" has no attribute "update"  [attr-defined]
```

can also add a `type: ignore` but really `mypy` is right here, this could cause a run time error